symplyphysics/laws/electricity/transmission_lines/transmission_matrix_of_π_type_circuit.py

```diff
@@ -0,0 +1,100 @@
+
+
+law = Eq(Matrix([[parameter_voltage_to_voltage, parameter_impedance], [parameter_conductance, parameter_current_to_current]]),
+         Matrix([[1 + third_impedance / second_impedance, third_impedance], [(1 / first_impedance) + (1 / second_impedance) + third_impedance / (first_impedance * second_impedance), 1 + third_impedance / first_impedance]]))
+
+# This law might be derived via "transmission_matrix_for_a_series_load_in_line" law and
+# "transmission_matrix_for_a_parallel_load_in_line" law.
+
+parallel_law_applied_1 = parallel_law.law.subs({
+    parallel_law.load_impedance: first_impedance,
+})
+matrix_derived_1 = solve(parallel_law_applied_1, [parallel_law.parameter_voltage_to_voltage, parallel_law.parameter_impedance, parallel_law.parameter_conductance, parallel_law.parameter_current_to_current], dict=True)[0]
+matrix_derived_1[parallel_law.parameter_impedance] = matrix_derived_1[parallel_law.parameter_impedance].scale_factor
+matrix_derived_1 = Matrix([[matrix_derived_1[parallel_law.parameter_voltage_to_voltage], matrix_derived_1[parallel_law.parameter_impedance]], [matrix_derived_1[parallel_law.parameter_conductance], matrix_derived_1[parallel_law.parameter_current_to_current]]])
+
+series_law_applied = series_law.law.subs({
+    series_law.load_impedance: third_impedance,
+})
+matrix_derived_3 = solve(series_law_applied, [series_law.parameter_voltage_to_voltage, series_law.parameter_impedance, series_law.parameter_conductance, series_law.parameter_current_to_current], dict=True)[0]
+matrix_derived_3[series_law.parameter_conductance] = matrix_derived_3[series_law.parameter_conductance].scale_factor
+matrix_derived_3 = Matrix([[matrix_derived_3[series_law.parameter_voltage_to_voltage], matrix_derived_3[series_law.parameter_impedance]], [matrix_derived_3[series_law.parameter_conductance], matrix_derived_3[series_law.parameter_current_to_current]]])
+
+parallel_law_applied_2 = parallel_law.law.subs({
+    parallel_law.load_impedance: second_impedance,
+})
+matrix_derived_2 = solve(parallel_law_applied_2, [parallel_law.parameter_voltage_to_voltage, parallel_law.parameter_impedance, parallel_law.parameter_conductance, parallel_law.parameter_current_to_current], dict=True)[0]
+matrix_derived_2[parallel_law.parameter_impedance] = matrix_derived_2[parallel_law.parameter_impedance].scale_factor
+matrix_derived_2 = Matrix([[matrix_derived_2[parallel_law.parameter_voltage_to_voltage], matrix_derived_2[parallel_law.parameter_impedance]], [matrix_derived_2[parallel_law.parameter_conductance], matrix_derived_2[parallel_law.parameter_current_to_current]]])
+
+matrix_derived = matrix_derived_1 * matrix_derived_3 * matrix_derived_2
+
+# Check if derived ABCD-parameters are same as declared.
+assert expr_equals(matrix_derived[0], law.rhs[0])
+assert expr_equals(matrix_derived[1], law.rhs[1])
+assert expr_equals(matrix_derived[2], law.rhs[2])
+assert expr_equals(matrix_derived[3], law.rhs[3])
```

Comment:
Use two-dimensional matrix access: `law.rhs[0][0]` and so on.

Reply:
> Use two-dimensional matrix access: `law.rhs[0][0]` and so on.

It doesn't work. An error appears:
    assert expr_equals(law.rhs[0][0], matrix_derived[0][0])
                       ~~~~~~~~~~^^^
TypeError: 'Add' object is not subscriptable

Reply:
Use `law.rhs[0, 0]`